mobile/src/main/java/org/openhab/habdroid/ui/OpenHABMainActivity.java

```diff
@@ -463,7 +449,8 @@ private void processIntent(Intent intent) {
         Log.d(TAG, "Got intent: " + intent);
         if (NfcAdapter.ACTION_NDEF_DISCOVERED.equals(intent.getAction())) {
             onNfcTag(intent.getData());
-        } else if (GcmIntentService.ACTION_NOTIFICATION_SELECTED.equals(intent.getAction())) {
+        } else if (ACTION_NOTIFICATION_SELECTED.equals(intent.getAction())) {
```

Comment:
no else-if needed, which is, however, maybe a task for somewhere else, as there're more branches, too. But, if you've time :D

Reply:
Maybe not strictly needed, but perfectly fine here as it's an either-or condition. One could even switch() on the action.